cluster/gke/util.sh

```diff
@@ -181,6 +181,11 @@ function kube-up() {
 
   # Bring up the cluster.
   "${GCLOUD}" ${CMD_GROUP:-} container clusters create "${CLUSTER_NAME}" "${create_args[@]}"
+
+  if [[ ! -z "${HEAPSTER_MACHINE:-}" ]]; then
```

Comment:
Seems like a fine change. I've been avoiding it because it makes the test environment more synthetic, but I guess it we're okay avoiding the quota I can finish it today.